jmclient/jmclient/cryptoengine.py

```diff
@@ -283,8 +284,53 @@ def sign_transaction(cls, tx, index, privkey, amount,
         return btc.sign(btc.serialize(tx), index, privkey,
                         hashcode=hashcode, amount=amount, native=True)
 
+class BTC_Timelocked_P2WSH(BTCEngine):
+
+    """
+    In this class many instances of "privkey" or "pubkey" are actually tuples
+    of (privkey, timelock) or (pubkey, timelock)
+    """
+
+    @classproperty
+    def VBYTE(cls):
+        return 100 #bech32 address vbyte
```

Comment:
Note that as per the comment around line 261 or so in this file, the "100" value was just something I ended up using as an artificial flag to specify specifically a *regtest* bech32 address (otherwise we can use the two character "human readable part" hrp tb, bc). It wasn't something I got from somewhere else, and as per that referenced comment, the idea is you could use the existing network byte and convert (0, 5 - mainnet, else testnet, unless "100" meaning regtest). If you hardcode 100 it's just always regtest. This is because pre-native-segwit, regtest behaved transparently like testnet, but not for bech32 addresses.
 #536 means none of this is necessary, it handles all this stuff (actually that's one of its best features).